idunn/api/utils.py

```diff
@@ -28,17 +30,88 @@
 from idunn.utils.index_names import INDICES
 from idunn.utils.es_wrapper import get_elasticsearch
 from idunn.places.exceptions import PlaceNotFound
+from idunn.utils.settings import _load_yaml_file
 
 logger = logging.getLogger(__name__)
 
-LONG = "long"
-SHORT = "short"
-LIST = "list"
-DEFAULT_VERBOSITY = LONG
-DEFAULT_VERBOSITY_LIST = LIST
+
+class Type(str, Enum):
+    # City = "city" # this field is available in Bragi but deprecated
+    House = "house"
+    Poi = "poi"
+    StopArea = "public_transport:stop_area"
+    Street = "street"
+    Zone = "zone"
+
+
+def get_categories():
+    categories_path = os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), "../utils/categories.yml"
+    )
+    return _load_yaml_file(categories_path)["categories"]
+
+
+def get_outing_types():
+    outing_types_path = os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), "../utils/categories.yml"
+    )
+    return _load_yaml_file(outing_types_path)["outing_types"]
+
+
+ALL_CATEGORIES = get_categories()
+ALL_OUTING_CATEGORIES = get_outing_types()
+
+
+class CategoryEnum(str):
+    """
+    Methods defining the behavior of the enum `Category` defined bellow.
+    """
+
+    def match_brand(self):
+        return ALL_CATEGORIES[self].get("match_brand", False)
+
+    def pj_filters(self):
+        return ALL_CATEGORIES[self].get("pj_filters")
```

Comment:
`pj_what` also needs to be defined similarly.  
"idunn/datasources/pages_jaunes.py" is currently using `c["pj_what"]` and is broken.